oslabs-beta/KataLog
# Tkp/frontend

##Overview
**Issue Type**
- [] Bug
- []Tech debt

**Description**
Added functionality for rendering the parsed logs.

**Steps to Reproduce Bug/Validate Feature/Confirm Tech Debt Fix**
Go to the root homepage
Click on nothing
Scroll down to the rendering of the logs
See the parsed logs separated by column headers

**Previous Behavior**
Rendering of logs didn't exist prior.

**Expected Behavior**
A complete rendering of the log data. 

**How Has This Been Tested**
Rendering on client side

**Screenshots & Videos**
N/A 

**Additional Context**
N/A
